gym/envs/registration.py

```diff
@@ -77,6 +77,10 @@ def make(self, **kwargs):
 
         return env
 
+    @property
+    def entry_point(self):
+        return self._entry_point
```

Comment:
if we were to just rename `_entry_point` field to `entry_point`, would that not do the trick?

Reply:
Sure! Wasn't sure what convention you wanted. Thought leaving it this way could imply to an onlooker that `_entry_point` isn't meant to be set apart from the construction of `EnvSpec`.

Reply:
There is also tests and perhaps some people already using `_entry_point`. It's not a problem to change the tests to use `entry_point`.

Reply:
If someone outside gym is accessing `_entry_point`, that's basically guaranteed to break at some point.

Reply:
Sounds good @christopherhesse. I update the PR with these changes.